File: light-bulb.py
total_expenses = 61
lamps_room1 = 10
days_room1 = 21
hours_room1 = 12
lamps_room2 = 20
days_room2 = 30
hours_room2 = 8

total_hours = (lamps_room1 * days_room1 * hours_room1) + (lamps_room2 * days_room2 * hours_room2)

room1_expenses = (lamps_room1 * days_room1 * hours_room1) / total_hours * total_expenses
room2_expenses = (lamps_room2 * days_room2 * hours_room2) / total_hours * total_expenses

print("Расходы на освещение первого помещения:", room1_expenses, "руб.")
print("Расходы на освещение второго помещения:", room2_expenses, "руб.")

File: light-bulb.php
<?php
$totalExpenses = 61;
$lampsRoom1 = 10;
$daysRoom1 = 21;
$hoursRoom1 = 12;
$lampsRoom2 = 20;
$daysRoom2 = 30;
$hoursRoom2 = 8;

$totalHours = ($lampsRoom1 * $daysRoom1 * $hoursRoom1) + ($lampsRoom2 * $daysRoom2 * $hoursRoom2);

$room1Expenses = ($lampsRoom1 * $daysRoom1 * $hoursRoom1) / $totalHours * $totalExpenses;
$room2Expenses = ($lampsRoom2 * $daysRoom2 * $hoursRoom2) / $totalHours * $totalExpenses;

echo "Расходы на освещение первого помещения: " . $room1Expenses . " руб.";
echo "Расходы на освещение второго помещения: " . $room2Expenses . " руб.";
?>
